src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -181,12 +204,22 @@ const GeneExternalReferences = () => {
   );
 };
 
-const RenderTranscriptXrefGroup = (props: { transcript: Transcript }) => {
+const RenderTranscriptXrefs = (props: { transcript: Transcript }) => {
```

Comment:
If this is a component, let's name it using a noun not a verb.

Reply:
That's done